packages/devtools_app/test/test_infra/scenes/standalone_ui/mock_editor_widget.dart

```diff
@@ -111,6 +135,24 @@ class _MockEditorWidgetState extends State<MockEditorWidget> {
                     'Use these buttons to simulate actions that would usually occur in the IDE.',
                   ),
                   const SizedBox(height: defaultSpacing),
+                  Row(
+                    children: [
+                      const Text('Editor: '),
+                      ElevatedButton(
+                        onPressed: editor.connected
+                            ? null
+                            : _withUpdate(editor.connectEditor),
+                        child: const Text('Connect'),
+                      ),
+                      ElevatedButton(
```

Comment:
super nit: add `const SizedBox(width: denseSpacing)` between these buttons

Reply:
Done in e07fca26a (here and below)